internal/armada/metrics/metrics.go

```diff
@@ -61,13 +64,118 @@ var queueResourcesDesc = prometheus.NewDesc(
 	nil,
 )
 
+var minQueueResourcesDesc = prometheus.NewDesc(
```

Comment:
Wondering, would it make sense to use a Prometheus histogram rather than multiple metrics for each interested quantile?